### DISCUSSION TITLE
Language Leader for Swiss German

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#801] [Language Leaders]
Hi @c0ffeeca7, would you be interested in being a language leader for Swiss German? :slightly_smiling_face:

**Comment by c0ffeeca7** [comment]
Sure! Thank you for accepting Swiss German as a language :-) 
Although there is no Swiss German - there is Bernese, Zürich German, Obwald German, Lower Wallis German, St. Gallen German, Basel German,...

**Comment by synesthesiam** [comment]
@dontinelli Would you be interested in being a language leader for `de-CH`?

**Reply by dontinelli** [reply]
> @dontinelli Would you be interested in being a language leader for `de-CH`?

Sure, pleased to support

**Reply by synesthesiam** [reply]
Great, thanks! Adding you now.

**Comment by synesthesiam** [comment]
Is anyone interested in [adding Swiss German support to Piper](https://github.com/rhasspy/piper/issues/49)? This would enable [local text to speech](https://www.home-assistant.io/blog/2023/04/27/year-of-the-voice-chapter-2/#piper-our-new-model-for-high-quality-local-text-to-speech) in Home Assistant for this language.

**Reply by dontinelli** [reply]
Thank you for making me aware, happy to look into this.

**Reply by Roemer** [reply]
I'd love to help here / take over but I miss some hints on what needs to be done.

**Reply by dontinelli** [reply]
> I'd love to help here / take over but I miss some hints on what needs to be done.

Any support is appreciated. I started a draft PR but have not come beyond this for the time being: https://github.com/espeak-ng/espeak-ng/pull/1724
